lib/resolvers/waifutcg.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+import Resolver from '../resolver';
+
+const RARITY_NAMES = ['common', 'uncommon', 'rare', 'super', 'ultra', 'legendary', 'mythical', 'god', 'special', 'promo'];
+
+export default class WaifuTCG extends Resolver {
+	transformURL(url, ctx) {
+		ctx.type = url.pathname.replace('/', '');
+		ctx.tcguser = url.searchParams.get('user');
+		switch (url.pathname) {
+
+			case '/booster':
+				ctx.cache_key = false; // booster Data is highly volatile and should not be cached
```

Comment:
Just so you know, there will always be some degree of caching happening with how FFZ's architecture currently works. In the future it might be possible to have uncached embeds or to cache bust them, just not something we can handle currently.

Reply:
Alright, this was one key I wasn't really sure about, thanks for the clarification!

I'll remove the explicit set to false then to avoid issues.

Reply:
Addressed with ea4cf7a